neqo-transport/src/packet.rs

```diff
@@ -37,6 +37,7 @@ const SAMPLE_SIZE: usize = 16;
 const SAMPLE_OFFSET: usize = 4;
 
 pub type PacketNumber = u64;
+pub type Version = u32;
```

Comment:
Do we still need this as a public value?  Or can it be private?

Reply:
it can be private if `QuicVersion` moves to packet.rs, which I've done.